src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/config.properties

```diff
@@ -1,3 +1,3 @@
 blurb=\
-    This launch method relies on DCOM and is often associated with <a href="https://wiki.jenkins-ci.org/display/JENKINS/Windows+slaves+fail+to+start+via+DCOM" target="_blank">subtle problems</a>. \
+    This launch method relies on DCOM and is often associated with <a href="https://wiki.jenkins-ci.org/display/JENKINS/Windows+agents+fail+to+start+via+DCOM" target="_blank">subtle problems</a>. \
```

Comment:
Wiki page has been already renamed, so we want to land it ASAP I'd guess

Reply:
I will fix it in a follow-up. Now it is https://github.com/jenkinsci/windows-slaves-plugin/blob/master/docs/troubleshooting.adoc